axelrod/graph.py

```diff
@@ -0,0 +1,126 @@
+"""
+Weighted undirected sparse graphs.
+
+Original source:
+https://github.com/marcharper/stationary/blob/master/stationary/utils/graph.py
```

Comment:
In order to get the `__repr__` method to output the list of edges I expect we could use an `edges` attribute or otherwise? That could be a useful attribute to have anyway?